src/ILCompiler.TypeSystem/tests/ILTestAssembly/UnificationGroup.il

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.class private auto ansi beforefieldinit UnificationGroup.MyBase
```

Comment:
Can you put this into `VirtualFunctionOverride.il` and in the VirtualFunctionOverride namespace?

If we ever get rid of unification groups, this test will be still relevant.